app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,32 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+  <%= f.text_field :name, autofocus: true, autocomplete: "name", placeholder: "Username" %>
+</div>
+
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
```

Comment:
| @Roland-Ntwali Design | Microverse Design |
|-|-|
| ![image](https://github.com/Roland-Ntwali/Blog-App/assets/93039359/8bd7f36a-5bc2-4fe7-813e-62c6e0406317) | ![image](https://github.com/Roland-Ntwali/Blog-App/assets/93039359/03496676-1f3f-4fb8-9639-88f40095c561) |

- [ ] Great work on putting so much effort into the functionality part 👍 but I guess it would be nice if you follow the project requirements for the design because in your design all the input fields are not consistent, not equally aligned to each other 😨 Kindly fix this issue by removing the labels and putting placeholders instead of them to make your design look professional 😊
- [ ] Second, kindly make an email placeholder for the email text field 🤔 because right now you have a password placeholder for the email input field 😨 Kindly fix it because it makes us confused while registering into your app 🚫